cardano-testnet/test/Test/Cli/Babbage/LeadershipSchedule.hs

```diff
@@ -0,0 +1,194 @@
+{-# LANGUAGE BlockArguments #-}
+{-# LANGUAGE DisambiguateRecordFields #-}
+{-# LANGUAGE NamedFieldPuns #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
+
+{-# OPTIONS_GHC -Wno-unrecognised-pragmas #-}
+
+{- HLINT ignore "Redundant id" -}
+{- HLINT ignore "Redundant return" -}
+{- HLINT ignore "Use head" -}
+{- HLINT ignore "Use let" -}
+
+module Test.Cli.Babbage.LeadershipSchedule
+  ( hprop_leadershipSchedule
+  ) where
+
+import           Cardano.CLI.Shelley.Output (QueryTipLocalStateOutput (..))
+import           Control.Monad (void)
+import           Data.List ((\\))
+import           Data.Monoid (Last (..))
+import           GHC.Stack (callStack)
+import           Hedgehog (Property)
+import           Prelude
+import           System.Environment (getEnvironment)
+import           System.FilePath ((</>))
+
+import qualified Data.Aeson as J
+import qualified Data.Aeson.Types as J
+import qualified Data.List as L
+import qualified Data.Time.Clock as DTC
+import qualified Hedgehog as H
+import qualified Hedgehog.Extras.Stock.IO.Network.Sprocket as IO
+import qualified Hedgehog.Extras.Test.Base as H
+import qualified Hedgehog.Extras.Test.File as H
+import qualified Hedgehog.Extras.Test.Process as H
+import qualified System.Directory as IO
+import qualified System.Info as SYS
+import qualified Testnet.Util.Base as H
+
+import           Cardano.Testnet
+import           Testnet.Util.Assert
+import           Testnet.Util.Process
+import           Testnet.Util.Runtime
+
+hprop_leadershipSchedule :: Property
+hprop_leadershipSchedule = H.integrationRetryWorkspace 2 "babbage-leadership-schedule" $ \tempAbsBasePath' -> do
+  H.note_ SYS.os
+  base <- H.note =<< H.noteIO . IO.canonicalizePath =<< H.getProjectBase
+  configurationTemplate <- H.noteShow $ base </> "configuration/defaults/byron-mainnet/configuration.yaml"
```

Comment:
`configuration/defaults/byron-mainnet/configuration.yaml` is the configuration file we alluded to earlier that is found in the git repository.

```
51 ┃   configurationTemplate <- H.noteShow $ base </> "configuration/defaults/byron-mainnet/configuration.yaml"
                ┃   │ "/Users/jky/wrk/iohk/cardano-node/configuration/defaults/byron-mainnet/configuration.yaml"
```

We use `noteShow` here which is the same as `note` except that the argument can be any value that has a `Show` argument.

We could have actually just used `note` because the argument is a `String`.